source/write_docs.rst

```diff
@@ -1,7 +1,57 @@
-Writing and compiling the docs
-""""""""""""""""""""""""""""""
+Writing documentation
+"""""""""""""""""""""
 
-We use sphinx to build the documentation. To build the documentation locally, the best is to create
+All code should be documented. Integration with readthedocs is an easy an effective way
```

Comment:
typo: ... is an easy **and** effective way ...

Reply:
fixed in b984525a0444f512883030a031a977bd950bd908